--- hatari-2.3.1/CMakeLists.txt.orig	2020-12-26 22:50:12.000000000 +0100
+++ hatari-2.3.1/CMakeLists.txt	2021-01-11 19:36:50.048045349 +0100
@@ -311,10 +311,6 @@
 	set(MANDIR share/man/man1)
 endif()
 
-if(NOT DOCDIR)
-	set(DOCDIR share/doc/hatari)
-endif()
-
 if(NOT ETCDIR)
 	if(WIN32)
 		set(ETCDIR .)
@@ -342,7 +338,6 @@
 		${CMAKE_BINARY_DIR}/config.h)
 
 add_subdirectory(src)
-add_subdirectory(doc)
 add_subdirectory(tests)
 add_subdirectory(tools)
 
