--- hatari-1.4.0/CMakeLists.txt~	2010-06-12 13:04:40.000000000 +0200
+++ hatari-1.4.0/CMakeLists.txt	2010-06-14 15:05:28.646457915 +0200
@@ -180,10 +180,6 @@
 	set(MANDIR share/man/man1)
 endif()
 
-if(NOT DOCDIR)
-	set(DOCDIR share/doc/hatari)
-endif()
-
 # #########################################
 # Create config.h and recurse into subdirs:
 # #########################################
@@ -192,7 +188,6 @@
 		${CMAKE_BINARY_DIR}/config.h)
 
 add_subdirectory(src)
-add_subdirectory(doc)
 add_subdirectory(tools)
 
 include(FindPythonInterp)
