--- hatari-1.5.0/CMakeLists.txt.orig	2011-07-19 00:32:26.000000000 +0200
+++ hatari-1.5.0/CMakeLists.txt	2011-07-23 09:53:31.982725700 +0200
@@ -220,10 +220,6 @@
 	set(MANDIR share/man/man1)
 endif()
 
-if(NOT DOCDIR)
-	set(DOCDIR share/doc/hatari)
-endif()
-
 if(NOT ETCDIR)
 	if(WIN32)
 		set(ETCDIR .)
@@ -241,7 +237,6 @@
 		${CMAKE_BINARY_DIR}/config.h)
 
 add_subdirectory(src)
-add_subdirectory(doc)
 add_subdirectory(tools)
 
 include(FindPythonInterp)
