--- hatari-1.1.0/Makefile.orig	2008-11-02 17:27:27.000000000 +0100
+++ hatari-1.1.0/Makefile	2008-12-14 23:57:47.639318667 +0100
@@ -59,8 +59,6 @@
 	$(INSTALL) -d $(DESTDIR)$(BINDIR)
 	$(INSTALL) -d $(DESTDIR)$(DATADIR)
 	$(INSTALL) -d $(DESTDIR)$(MANDIR)
-	$(INSTALL) -d $(DESTDIR)$(DOCDIR)
-	$(INSTALL) -d $(DESTDIR)$(DOCDIR)/images
 	$(INSTALL_PROGRAM) src/hatari $(DESTDIR)$(BINDIR)/hatari
 	$(INSTALL_PROGRAM) tools/hmsa/hmsa $(DESTDIR)$(BINDIR)/hmsa
 	$(INSTALL_DATA) src/hatari-icon.bmp $(DESTDIR)$(DATADIR)/hatari-icon.bmp
@@ -68,5 +66,3 @@
 	  $(INSTALL_DATA) src/tos.img $(DESTDIR)$(DATADIR)/tos.img ; \
 	fi
 	$(INSTALL_DATA) hatari.1.gz $(DESTDIR)$(MANDIR)/
-	$(INSTALL_DATA) doc/*.txt doc/*.html $(DESTDIR)$(DOCDIR)/
-	$(INSTALL_DATA) doc/images/*.png $(DESTDIR)$(DOCDIR)/images/
