--- hatari-2.3.1/share/applications/hatari.desktop.orig	2021-01-11 19:37:11.294596914 +0100
+++ hatari-2.3.1/share/applications/hatari.desktop	2021-01-11 19:40:28.446862182 +0100
@@ -6,6 +6,6 @@
 Exec=hatari %f
 Icon=hatari
 MimeType=application/x-st-disk-image;application/vnd.msa-disk-image;application/vnd.fastcopy-disk-image;application/x-stx-disk-image;
-Categories=Game;Emulator;
+Categories=Utility;Emulator;
 Keywords=Emulator;Atari;Atari ST;Atari Falcon;
 Terminal=false
--- hatari-2.3.1/python-ui/hatariui.desktop.orig	2021-01-11 19:37:11.294596914 +0100
+++ hatari-2.3.1/python-ui/hatariui.desktop	2021-01-11 19:41:11.523295483 +0100
@@ -6,6 +6,6 @@
 Exec=hatariui
 Icon=hatari
 StartupNotify=true
-Categories=Game;Emulator;
+Categories=Utility;Emulator;
 Keywords=Emulator;Atari;Atari ST;Atari Falcon;
 Terminal=false
