--- hatari-2.5.0/CMakeLists.txt.orig	2024-04-18 22:01:11.000000000 +0200
+++ hatari-2.5.0/CMakeLists.txt	2024-09-22 14:42:25.451059767 +0200
@@ -329,10 +329,6 @@ if(NOT MANDIR)
 	set(MANDIR share/man/man1)
 endif()
 
-if(NOT DOCDIR)
-	set(DOCDIR share/doc/hatari)
-endif()
-
 if(NOT ETCDIR)
 	if(WIN32)
 		set(ETCDIR .)
@@ -360,7 +356,6 @@ configure_file(${CMAKE_SOURCE_DIR}/cmake
 		${CMAKE_BINARY_DIR}/config.h)
 
 add_subdirectory(src)
-add_subdirectory(doc)
 add_subdirectory(tools)
 
 if(NOT CMAKE_CROSSCOMPILING)
